apps/chat/src/components/Marketplace/SearchHeader.tsx

```diff
@@ -43,7 +46,7 @@ const AddAppButton = ({ menuItems }: AddAppButtonProps) => {
     return (
       <button
         onClick={visibleActions[0].onClick}
-        className="button button-primary hidden items-center gap-3 md:flex"
+        className="button button-primary hidden items-center gap-2 md:flex"
```

Comment:
```suggestion
        className="button button-primary hidden items-center md:flex"
```

gap-2 already in the .button class